test/lib/postgres.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+const assert = require('assert').strict;
+const {PostgresDatabase} = require('../../lib');
+
+function testMod(mod) {
+	try {
+		require(mod);
+	} catch (e) {
+		return it.skip;
+	}
+	return it;
+}
+
+// only run these if you already have postgres configured
+describe.skip("Postgres features", () => {
+	it("Should be able to connect to a database", async () => {
+		this.database = new PostgresDatabase();
+	});
+	it("Should be able to insert data", async () => {
+		assert.doesNotThrowAsync(async () => {
+			await this.database.query(`CREATE TABLE test (col TEXT, col2 TEXT)`);
+			await this.database.query(
+				`INSERT INTO test (col, col2) VALUES ($1, $2)`,
+				['foo', 'bar'],
+			);
+		});
+	});
+	testMod('sql-template-strings')('Should support sql-template-strings', async () => {
+		assert.doesNotThrowAsync(async () => {
```

Comment:
These should be awaited.

Reply:
Oops.

Reply:
You resolved this, but did not actually fix the bug.

Reply:
This is done.